manager/testdata/new-blueprints/scenario2/plotter.yaml

```diff
@@ -120,12 +108,6 @@ spec:
           parameters:
             source:
               assetId: "m4d-notebook-sample/inventory"
-            api:
```

Comment:
Why is the api field removed? Where in this current plotter is defined that the read service should offer the given asset using the arrow-flight protocol with a given asset id?

Reply:
The read access here is done by the workload. Nothing is done in the deployment.
In a different scenario where another module reads the data from the read service, the module gets this information in its params (e.g. source).